--- configure.ac~	2011-03-15 10:34:41.000000000 +0000
+++ configure.ac	2011-04-14 18:13:19.365888483 +0100
@@ -15,8 +15,8 @@
 fi
 
 dnl figure out the libssh2 version
-VERSION=`$SED -ne 's/^#define LIBSSH2_VERSION *"\(.*\)"/\1/p' ${srcdir}/include/libssh2.h`
 AM_INIT_AUTOMAKE
+VERSION=`$SED -ne 's/^#define LIBSSH2_VERSION *"\(.*\)"/\1/p' ${srcdir}/include/libssh2.h`
 AC_MSG_CHECKING([libssh2 version])
 AC_MSG_RESULT($VERSION)
 
